purview/models/freshmaker.py

```diff
@@ -1,11 +1,11 @@
 from neomodel import (
-    StructuredNode, UniqueIdProperty, RelationshipFrom, RelationshipTo,
```

Comment:
In the future, the relationship fixes should be in a separate commit.

Reply:
Yeap will keep that in mind!